apps/files/src/helpers/collaboratorRolesDefinition.js

```diff
@@ -7,33 +7,33 @@ function returnOriginal (string) {
 /**
    * Returns object with collaborator roles
    * @param {boolean} isFolder  Defines if the item is folder
-   * @param {function} $gettext  Function to translate neccessary strings
```

Comment:
Bring this back pls. Translate wouldn't export strings to transifex. Already did that once myself :grin:

Reply:
Whoops, sorry 😅  But the way it was wasn't correct either: function didn't pick up the $gettext object resulting in missing translation. Will look into it again.

Reply:
Ok, thanks for taking care

Reply:
Done. ;-) Fyi, the function call has to look like this: https://github.com/owncloud/phoenix/blob/b9bcc54c31ae8b7b87b9263332be118cb24d1ad8/apps/files/src/mixins/collaborators.js#L45

Previously, the function expected an object by the key `$gettext` but an object with the key `translate` was provided, so it was just ignored. That was the reason why I changed the key in the function to `translate`. As it needs to be called `$gettext` I changed it back and instead made sure that the object key is `$gettext` (and not `translate` anymore). See link above. Ping me if this needs better explanation. :-)